390/starry_zhou.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "MP2_EDA"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
```

Comment:
let's set `warning = FALSE` to all our code blocks so that the .html output is a little less jarring